component/main.jsonnet

```diff
@@ -57,7 +57,12 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
   rules+: [
     {
       apiGroups: [ '' ],
-      resources: [ 'pods', 'pods/log', 'pods/status', 'events', 'services' ],
+      resources: [ 'pods', 'pods/log', 'pods/status', 'events', 'services', 'namespaces' ],
+      verbs: [ 'get', 'list', 'watch' ],
+    },
+    {
+      apiGroups: [ 'apps' ],
+      resources: [ 'statefulsets' ],
```

Comment:
This is so that the database shows up in the `Topology` tab. I think this should be fine as we don't really leak more information than we do through the pod anyway